src/app/zap-templates/zcl/data-model/chip/bridged-device-basic-information.xml

```diff
@@ -97,7 +97,8 @@ limitations under the License.
 
         <command source="client" code="0x80" name="KeepActive" optional="true" apiMaturity="provisional">
           <description> The server SHALL attempt to keep the devices specified active for StayActiveDuration milliseconds when they are next active.</description>
-          <arg name="StayActiveDuration" type="int32u"/>
+          <arg id="0" name="StayActiveDuration" type="int32u"/>
+          <arg id="1" name="TimeoutMs" type="int32u"/>
```

Comment:
I'm not totally sure what this GETS you since you're using the access interface (possibly some zap gui assistance?) but this tag supports min and max fields as well, and those should probably be specified.

Reply:
TimeoutMs? It allows the cert test go from taking 2 hours to less than 5mins

Reply:
Oh... You are referring to specifying min/max constraints in the XML. I will add soon

Reply:
+1, we need to add 'min' and 'max' field to align with the spec defination

Reply:
ZAP does not track min/max for command fields, as far as I know, by the way.

Reply:
>ZAP does not track min/max for command fields, as far as I know, by the way.

That is why I didn't add it in. But I will add it just for clarity and to resolve this comment